test/e2e/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # We need to build in a Linux environment to support C libraries, e.g. RocksDB.
 # We use Debian instead of Alpine, so that we can use binary database packages
 # instead of spending time compiling them.
-FROM cometbft/cometbft-db-testing:v0.14.2
+FROM cometbft/cometbft-db-testing:latest
```

Comment:
I had to switch to the standard golang one to get this to work. https://github.com/celestiaorg/celestia-core/pull/1609

Reply:
is this a reason to use latest instead of the exact version? With latest, we might not be aware when there is a new version that breaks the tests